app/src/main/java/io/appium/settings/RecorderService.java

```diff
@@ -0,0 +1,135 @@
+package io.appium.settings;
+
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.media.projection.MediaProjection;
+import android.media.projection.MediaProjectionManager;
+import android.os.Build;
+import android.os.IBinder;
+import android.util.DisplayMetrics;
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+import io.appium.settings.helpers.NotificationHelpers;
+
+public class RecorderService extends Service {
+    private static final String TAG = "RecorderService";
+
+    public static final int REQUEST_CODE_SCREEN_CAPTURE = 123;
+    private static final String BASE = "io.appium.settings.recording";
```

Comment:
can we read the package name dynamically instead of hardcoding it?

Reply:
it is possible yeah but we have 2 choices AFAIK, one of them is `context.getPackageName()` and other one is `BuildConfig.APPLICATION_ID` and since 1st one requires Context object and we dont have context in that code flow, i can apply 2nd method, is it fine for you ? 👍

Reply:
okay done https://github.com/sirmordred/io.appium.settings/commit/deff3af08b2254951afa8d28d34c2dcd0197f437